src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -992,6 +992,26 @@ class VariantSampleMatrix[T](val metadata: VariantMetadata,
         "sa" -> saSignature),
       Array("s"))
   }
+
+  def annotateVariantsKeyTable(kt: KeyTable, code: String) = {
+    val ktKeyTypes = kt.keySignature.fields.map(_.`type`)
+
+    if (ktKeyTypes.size != 1 || ktKeyTypes(0) != TVariant)
+      fatal(s"Key Signature of KeyTable must be 1 field with type `Variant'. Found `${ktKeyTypes.mkString(", ")}'")
```

Comment:
Should you print out the key signature here?  I feel like the column names could be useful.